stories/Timeline/Timeline.stories.js

```diff
@@ -0,0 +1,188 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { withA11y } from '@storybook/addon-a11y'
+import { withInfo } from '@storybook/addon-info'
+
+import Esempi from './docs/Esempi.md'
+
+import {
+  TimelineWrapper,
+  Card,
+  CardBody,
+  CardTitle,
+  CardText,
+  CardCategory,
+  CardSignature,
+  CardReadMore,
+  TimelinePin
+} from '../../src'
+
+const EsempiComponent = () => (
+  <div>
+    <TimelineWrapper>
+      <div className="row">
+        <div className="col-12">
```

Comment:
```jsx
<Row>
  <Col>
...
```

Try to stick with reactstrap components as much as possible to keep consistency into the stories.

Reply:
Oops! yes